src/components/timecard/edit-shift-hours/EditShiftHours.jsx

```diff
@@ -165,6 +233,9 @@ const EditShiftHours = ({
             .setTimeEntryId(responseItem.id)
             .setFinishNextDay(timeEntry.finishNextDay);
 
+          if (startEntryExists) {
```

Comment:
Is it possible to submit without a start entry? Not sure if this if statement is needed

Reply:
That is just there to stop notifications raising on new entries. It was easier to use that existing constant than creating a isNewRecord from it.